RodSoriano/nekupu
# Adder seeder setup

## Description

This PR includes several updates and enhancements:

- 🗃️ Updated models and added initial migration.
- ✨ Added seeders and factories for contacts, posts, comments, and categories.
- 🔥 Removed the deprecated `message` entity since chat will be handled in memory.
- 🏷️ Added `comment type` enum for better classification.
- 🌱 Updated seeders for `category` and `user` with improved logic.
- 🐛 Fixed user entity properties and resolved duplication in `JoinTable` for `Post` entity.
- 🎨 Applied code formatting and ordered imports for consistency.
- ✨ Introduced modules for `user`, `post`, `message`, `contact`, `comment`, and `category`.
- ✨ Added support for migrations and seeding.
- 📝 Updated the root README with seed instructions and migrations.
- ✨ Added `.eslintignore` to manage linting exclusions.

## Type of Change

Please indicate the type of change included in this PR:


## How to Test

1. Run the application using the root `dev` script.
2. Ensure all environment variables load correctly in the backend.
3. Run migrations and execute the seeders to populate the database.
4. Confirm that the API's code formatting adheres to the project standards.

## Verification Checklist

Please verify the following:

- [ ] Potential security issues
- [ ] Test coverage validation

## Suggested Reviewers

- @RodSoriano

## Labels

- `enhancement`
